deployment.yaml

```diff
@@ -12,3 +12,4 @@ secrets:
 env:
   PRODUCTION: true
   DEFAULT_TIMEZONE: "America/New_York"
+  BUGSNAG_ENABLED: false
```

Comment:
is this the right key, isn't it REACT_APP_ENABLE_BUGSNAG

Reply:
👀 think you're right